src/dotnet/commands/dotnet-new/FSharp_Console/$projectName$.fsproj

```diff
@@ -1,5 +1,4 @@
-﻿<Project ToolsVersion="15.0">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
+﻿<Project Sdk="FSharp.NET.Sdk;Microsoft.NET.Sdk" ToolsVersion="15.0">
```

Comment:
So if I understand this correctly, this brings in just a SDK.props\targets that overrides LanguageTargets and the actual compile targets are brought in by the PackageRef? Why this split between the SDK and PackageReference? Why don't we add the entire package to the Bundled SDK and not have the PackageReference in the project at all. You've made a reference to it in your PR description that remote SDKs are implemented yet but why don't we simulate the full F# SDK? Are there versioning concerns?